src/commands/AbstractStateModificationCommand.ts

```diff
@@ -77,4 +86,15 @@ export default abstract class AbstractStateModificationCommand extends Command {
         return `${this.stateModificationObjectType.substring(0, 1).toUpperCase()}${this.stateModificationObjectType.substring(1)}`;
     }
 
+    private handleExclusionsAndDevelopmentCommand (id: string | undefined, excludedServices: string[]):boolean {
+        if (id) {
+            const inExclusionMode = (/^exclusions:(add|remove)$/).test(id);
+            const inDevelopmentMode = (/^development:(enable|disable)$/).test(id);
+            const hasExcludedServices = excludedServices.length > 0;
+            return inExclusionMode || (hasExcludedServices && inDevelopmentMode);
+        }
+        return false;
```

Comment:
Might be easier to read if there was a single point of return so set a variable and then modify this to value of `inExclusionMode || (hasExcludedServices && inDevelopmentMode);` ?

Reply:
I will fix in the next PR.